sources/packages/web/src/types/AppRoutes.ts

```diff
@@ -47,6 +47,8 @@ export enum AppRoutes {
   InstitutionManageUsers = "manage-users",
   ActiveApplicationScholasticStandingView = "active-application/location/:locationId/scholastic-standing/:scholasticStandingId",
   OfferingsUpload = "offerings-upload",
+  InstitutionStudentDetail = "student-detail/:studentId",
```

Comment:
`student/:studentId` should be enough IMO

Reply:
Done for both institution and ministry.